BondageClub/Screens/Online/ChatRoom/ChatRoom.js

```diff
@@ -2747,15 +2747,15 @@ function ChatRoomStopLeave(){
 /**
  * Sends an administrative command to the server for the chat room from the character dialog.
  * @param {string} ActionType - Type of action performed.
- * @param {boolean} [Publish] - Whether or not the action should be published.
+ * @param {boolean | string} [Publish] - Whether or not the action should be published.
```

Comment:
I like to add this just to see what the default behaviour is without having to look through the code of the function.
```suggestion
 * @param {boolean | string} [Publish=true] - Whether or not the action should be published.
```